.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Mark or close stale issues and PRs'
+
+on:
+    schedule:
+        - cron: '00 * * * *'
+
+jobs:
+    stale:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/stale@v3.0.14
+              with:
+                  repo-token: ${{ secrets.GITHUB_TOKEN }}
+                  days-before-issue-stale: 90
+                  days-before-pr-stale: 30
+                  days-before-close: 3
+                  stale-issue-message: 'This issue has been automatically marked as stale because it has not had any recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
+                  stale-pr-message: 'This PR has been automatically marked as stale because it has not had any recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
+                  close-issue-message: 'This issue was automatically closed due to being stale. Feel free to open a new one if you still experience this problem.'
+                  close-pr-message: 'This PR was automatically closed due to being stale.'
+                  stale-pr-label: 'status: stale'
+                  stale-issue-label: 'status: stale'
+                  exempt-issue-labels: 'cooldown period,feature: activity panel,feature: analytics,feature: components,feature: customer effort score,feature: home screen,feature: inbox,feature: marketing,feature: navigation,feature: onboarding,feature: rest api,feature: settings,feature: setup checklist,feature: wcpay,priority: high,priority: critical'
```

Comment:
I'm not sure we need/should have the 'feature: ' labels exempt... just because an issue is (perhaps correctly) categorized doesn't mean it still isn't stale.

Reply:
@mattsherman Happy to remove this and see how we go. I'm changing to a dry run initially anyway so the risk is low.